nilearn/decoding/tests/test_decoder.py

```diff
@@ -326,6 +432,7 @@ def test_decoder_dummy_classifier_strategy_prior():
     X, y, mask = _make_binary_classification_test_data(n_samples=400)
```

Comment:
Can this and the next test below also use one of the binary classification data fixtures or do you need n_samples=400?

Reply:
Could only bring it down to 300 without having the test start failing.